packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ProjectInfo/Delete.tsx

```diff
@@ -27,17 +38,45 @@ export const Delete = () => {
   };
 
   return (
-    <Stack justify="center" marginBottom={3}>
-      <Button
-        // @ts-ignore
-        onClick={onDelete}
-        variant="link"
+    <Element>
+      <Stack justify="center" marginBottom={6}>
+        <Button
+          // @ts-ignore
+          onClick={onDelete}
+          variant="link"
+          css={css({
+            ':hover,:focus': { color: 'errorForeground' },
+          })}
+        >
+          {`Delete ${customTemplate ? `Template` : `Sandbox`}`}
+        </Button>
+      </Stack>
+      <Element
+        marginBottom={3}
         css={css({
-          ':hover,:focus': { color: 'errorForeground' },
+          paddingX: 2,
         })}
       >
-        {`Delete ${customTemplate ? `Template` : `Sandbox`}`}
-      </Button>
-    </Stack>
+        <Stack gap={2}>
+          {links.map(link => (
+            <Link
+              target="_blank"
+              rel="noopener noreferrer"
+              css={css({ color: 'sideBar.border' })}
+              href={link.href}
+            >
+              {link.icon}
```

Comment:
If you destructur `link` and rename `icon` to `Icon` and just put the component there, you could do

```suggestion
              <Icon />
```

Reply:
I tried that and there were some issues with it so I left it the way it is